run_tests.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Run tests for the Vector Drawing XBlock.
+
+This script is required to run our selenium tests inside the xblock-sdk workbench
+because the workbench SDK's settings file is not inside any python module.
+"""
+
+import os
+import logging
+import sys
+
+from django.conf import settings
+from django.core.management import execute_from_command_line
+
+logging_level_overrides = {
+    'workbench.views': logging.ERROR,
+    'django.request': logging.ERROR,
+    'workbench.runtime': logging.ERROR,
+}
+
+if __name__ == '__main__':
+    # Use the workbench settings file:
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'workbench.settings')
+    # Configure a range of ports in case the default port of 8081 is in use
+    os.environ.setdefault('DJANGO_LIVE_TEST_SERVER_ADDRESS', 'localhost:8081-8099')
+
+    try:
+        os.mkdir('var')
```

Comment:
@singingwolfboy The workbench uses the `var` directory [for database and log files](https://github.com/edx/xblock-sdk/commit/f8bff8ba9279ae5bcbd99eed0731f39273a237ae). Creating it here to ensure that the tests don't fail because it's missing.